package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contentful-management",
-  "version": "5.2.1",
+  "version": "5.12.0",
```

Comment:
I guess this is now maintained by semantic release, but I needed to bump this so that installing from git works correctly.